OasisDEX/oasis-borrow
# Asset page

# [Asset page](https://app.shortcut.com/oazo-apps/story/3231/create-assets-page)

  
## Changes 👷‍♀️
- Assets page created, based on config in `feature/content`
- Landing page pills, TODO
  
## How to test 🧪
- Navigate to `/asset/eth`
- Navigate to `/asset/lp-token`
- Test what happens when user inputs wrong url
- Pills on the landing page

* see all available options in `content.ts`
* enable new landing page in localstorage by setting the flag

## Missing:
- Content for description and links
    
## Definition of done ✔️
